components/dashboard/cells/ImplementationPhaseCell.tsx

```diff
@@ -0,0 +1,30 @@
+import { ActionListAction } from '../dashboard.types';
+import styled from 'styled-components';
+import { PlanContextFragment } from 'common/__generated__/graphql';
+import ActionPhase from 'components/actions/ActionPhase';
+
+interface Props {
```

Comment:
Would it be better to use `type` instead of `interface`? I bumped into this styleguide over the weekend, and they recommend `type OwnProps`: https://docs.twenty.com/developer/frontend/style-guide#props

They seemed to have pretty reasonable other practices, too. We might want to consider adopting some of them.

Reply:
I like this! We're following many of those already, and [others just make sense](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces). I did some reading up on `interface` vs `type` and felt this is the main benefit of types over interfaces:

- Interfaces with the same name in the same scope [merge declarations](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces), whereas duplicate types will throw an error. While rare, there's a possibility for bugs/ unpredictability if we accidentally declare duplicate interfaces.